src/components/ebay-icon/examples/all.marko

```diff
@@ -1554,6 +1554,12 @@ div.icon-examples
         span.text
             -- ebay-stepper-confirmation-icon
 
+   div
```

Comment:
Good use of Marko concise syntax here.... from somebody who isn't a fan of concise syntax ;-)

Reply:
I’m not a fan either but generating this file in concise syntax was much easier than doing html